executor/executor.go

```diff
@@ -289,19 +293,27 @@ func (executor *Executor) execute(
 		Containers: []kuber.ContainerResourcesRequirements{
 			{
 				Name: containerName,
-				Limits: kuber.RequestLimit{
-					Memory: decision.ContainerResources.Limits.Memory,
-					CPU:    decision.ContainerResources.Limits.CPU,
-				},
-				Requests: kuber.RequestLimit{
-					Memory: decision.ContainerResources.Requests.Memory,
-					CPU:    decision.ContainerResources.Requests.CPU,
-				},
+				Limits: kuber.RequestLimit{},
+				Requests: kuber.RequestLimit{},
```

Comment:
this is the definition of ContainerResourcesRequirements
```
type ContainerResourcesRequirements struct {
	Name     string
	Requests RequestLimit
	Limits   RequestLimit
}
```

which mean if the limits and requests lefts like that and there wasn't a limit for example it will set the limit to zeros, you should update the kube struct to be
```
type ContainerResourcesRequirements struct {
	Name     string
	Requests *RequestLimit `json:"requests,omitempty"`
	Limits   *RequestLimit `json:"limits,omitempty"`
}
```


and here int he implementation you should use **new(kuber.RequestLimit)**  to define the resources